solutions/bronze/usaco-1251.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: usaco-1251
+source: USACO Bronze 2022 December
+title: Cow College
+author: Chu Minjae
+---
+[Official Analysis (C++, Java, Python)](http://www.usaco.org/current/data/sol_prob1_bronze_dec22.html)
+
+## Explanation
+
+We can solve this problem by sorting the maximum tuition rates!
+
+There are two reasons for doing this:
+
+1. We need to find the smallest optimal tuition, so by searching the value from the smallest to the largest, we can get the smallest optimal tuition.
+
+2. We can easily calculate the current tuition from the current index.
+
+As we iterate through the list, we can calculate the total tuition at an index $i$ with the equation `maxTuition[i] * (totalNumberOfCows - i)`.
+
+Since we've already sorted the cows, we can quickly figure out how many people can pay the tuition from `totalNumberOfCows - i`. We'll take the maximum of these quantities.
+
+Note that for this question, 64-bit integers should be used as numbers can go up to `10^5 * 10^6 = 10^11`.
+
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N \log N)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+int main() {
+	int totalNumberOfCows;
+	int x;
+	int bestTuition = 0;
```

Comment:
put this above where you actually use it
also use snake_case instead of camelCase for names